includes/Checker/Check.php

```diff
@@ -27,4 +27,11 @@ interface Check {
 	 *                   the check).
 	 */
 	public function run( Check_Result $result );
+
+	/**
+	 * Gets the category for the check.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function get_category();
```

Comment:
Nit: all the new methods introduced in this PR should have proper `@return` annotations. Once we bump the PHPStan strictness level this will be mandatory.

Reply:
Thanks @swissspidy for the feedback. In https://github.com/10up/plugin-check/pull/220/commits/b6a5d2bd849c6e9047a56ea75d13926c877e6324 i have added missing return for the functions.